src/main/java/core/basesyntax/HelloWorld.java

```
@@ -1,9 +1,64 @@
+            String inputFile = "input.csv";
+            String outputFile = "output.csv";
+
+            Map<String, Integer> fruitShop = new HashMap<>();
+
+            try (BufferedReader read = new BufferedReader(new FileReader(inputFile))) {
+                String line;
+                while ((line = read.readLine()) != null) {
+                    String[] parts = line.split(",");
+                    String operation = parts[0].trim();
+                    int quantity = Integer.parseInt(parts[2].trim());
+
+                    int current = fruitShop.getOrDefault(operation, 0);
+
+                    switch (operation) {
+                        case "b":
+                            fruitShop.put(operation, quantity);
+                            break;
+                        case "s":
+                            fruitShop.put(operation, current + quantity);
+                            break;
+                        case "p":
+                            fruitShop.put(operation, current - quantity);
+                            break;
+                        case "r":
+                            fruitShop.put(operation, current + quantity);
+                            break;
+                    }
+                }
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(outputFile))) {
+                writer.write("fruit,quantity");
+                writer.newLine();
+
+                for (Map.Entry<String, Integer> entry : fruitShop.entrySet()) {
+                    String fruit = entry.getKey();
+                    int quantity = entry.getValue();
+                    writer.write(fruit + "," + quantity);
+                    writer.newLine();
+                }
+
+                System.out.println("Отчет успешно записан в файл " + outputFile);
+            } catch (IOException e) {
+                e.printStackTrace();
```

According to the checklist, you should not ignore exceptions in `try-catch` blocks. Instead of `e.printStackTrace()`, rethrow a `RuntimeException` with an informative message and the exception object. For example:
```java
throw new RuntimeException("Error writing to file", e);
```